File: generator/output/ts-model/wallet-log.model.ts
class WalletLogModel {
  public id: number = 0;

  // deleted 
  public deleted?: number = 0;

  // user_id 
  public user_id: number = 0;

  // money 资金变动
  public money: number = 0;

  // description 资金变动
  public description: string;

  public created_at: string;

  public updated_at: string;
}

const WalletLogModelFields = {
  id: "id",
  deleted: "deleted",
  user_id: "user_id",
  money: "money",
  description: "description",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {WalletLogModel, WalletLogModelFields};

File: generator/output/entityModel/WalletLogModel.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.String;
import java.lang.Long;
import java.math.BigDecimal;
import java.lang.Boolean;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* WalletLog
*/
@Data()
public class WalletLogModel {
  public Long id = null;

  /**
  * 
  */
  public Boolean deleted = null;

  /**
  * 
  */
  public Long user_id = null;

  /**
  * 资金变动
  */
  public BigDecimal money = null;

  /**
  * 资金变动
  */
  public String description = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}
